src/myrtlespeech/builders/pre_process_step.py

```diff
@@ -25,8 +27,14 @@ def build(
         :py:class:`ValueError`: On invalid configuration.
     """
     step_type = pre_process_step_cfg.WhichOneof("pre_process_step")
+    step: Callable
```

Comment:
Why is this `Callable` now required? If mypy was failing maybe the return type of `Union[MFCC, Standardize]` just needs updating to include `MFCCLegacy`?

Reply:
I updated the return type to `Union[AddContextFrames, MFCC, MFCCLegacy, SpecAugment, Standardize]` and deleted the `step: Callable` line but this still resulted in `mypy` errors. 

I think mypy dislikes the multiple types in the `if step_type == 'mfcc'` branch so I've put a type def there as  `step: Union[MFCC, MFCCLegacy]` and it now passes. 

I've left `Union[<all pp types>]` change as this is correct (though not the cause of the problem).